test/functional/test_framework/test_node.py

```diff
@@ -580,7 +580,8 @@ def send_cli(self, command=None, *args, **kwargs):
         if command is not None:
             p_args += [command]
         p_args += pos_args + named_args
-        self.log.debug("Running bitcoin-cli command: %s" % command)
+        options_to_log = '' if not self.options else ', options: ' + ', '.join(self.options)
+        self.log.debug("Running bitcoin-cli command: {}{}".format(command, options_to_log))
```

Comment:
Might as well just log p_args[2:] or so, because -named is an option as well

Reply:
Done, set before `p_args += [command]` to not log the command in the options as well.

Reply:
Do you think it's more useful to set it after ` p_args += pos_args + named_args` and remove the command from the list?

Reply:
```py
self.log.debug("Running bitcoin-cli {}".format(p_args[2:]))
```

should do all the things:

* Include whether it is `-named` or not
* Include all args, like `--tracerpc`
* Include the command

Reply:
Done